### ISSUE TITLE
Impossible to withdraw rewards from ledger wallet in Daedalus

### ISSUE THREAD

**Issue by kzebrackio** [input-output-hk/daedalus#2298]
I just tried to move my balance from my Daedalus Ledger wallet to another Daedalus wallet and encountered that my staking rewards cant be transferred. Only my UT/XO balance. I'm quite a big bag holder and my rewards are piling up. A couple of thousand ADA are waiting for me, but it annoys me they cannot be withdrawn. 

I know there's a little fee for withdrawing ada, but thats not the issue. As soon as i surpass my UT/XO balance, i get the message: "Not enough money to make this transaction", while my balance says i have thousands more ada.

Is this a bug. What if i wanna sell my bag at a certain point, how do i redeem these rewards?? Selling should be able to do quickly in this highly volatile market. I moved my bag to Daedalus to earn rewards, but if i cant withdraw them it seems kinda useless.

**Comment by kzebrackio** [comment]
Another issue is that my Daedalus Ledger wallet does not allow me to withdraw ada to an exchange wallet directly. If i enter an exchange wallet my ledger keeps waiting for commands. If i enter my other daedalus wallet account, the commands do send correctly. So now i need a double transfer to get my ada on an exchange. First to my other daedalus wallet, next to my exchange wallet. Also a bit weird.

**Comment by nikolaglumac** [comment]
Hi @kzebrackio!
Sorry to hear about the issues you are facing...

The first one is a known issue reported here: https://github.com/input-output-hk/daedalus/issues/2274 - please track it under that link (we are still waiting on a fix on the [cardano-wallet](https://github.com/input-output-hk/cardano-wallet) side). I can not give you an ETA on that one 😞 
If you can not wait for the fix, a workaround would be to use Yoroi Wallet.

The second one is also a known issue we are fixing at the moment. The fix should be included in the next Daedalus release (most likely this week).

Sorry for the trouble and thanks for the patience!

**Comment by nikolaglumac** [comment]
Hi @kzebrackio!

> Another issue is that my Daedalus Ledger wallet does not allow me to withdraw ada to an exchange wallet directly. If i enter an exchange wallet my ledger keeps waiting for commands. If i enter my other daedalus wallet address, the commands do send correctly. So now i need a double transfer to get my ada on an exchange. First to my other daedalus wallet, next to my exchange wallet. Also a bit weird.

☝️  A fix for this issue has been released in Daedalus Mainnet 3.2.0.

I am closing this ticket. You can follow the updates regarding the other issue you reported (unable to send/withdraw rewards on hardware wallets) here: https://github.com/input-output-hk/daedalus/issues/2274
